src/test/shell/bazel/bazel_cc_code_coverage_test.sh

```diff
@@ -376,6 +376,11 @@ function test_cc_test_coverage_gcov() {
       output_file_json="output_file.json"
       zcat $agcda $tgcda $dagcda > $output_file_json
 
+      # Remove all branch information from the json.
+      # We disable branch coverage for gcov 7 and it is easier to remove it
+      # than it is to put it back.
+      # Replace "branch": [..] with "branch": [] - (non-empty [..])
```

Comment:
nit: I assume you meant to write `branches` here?

Reply:
Fixed